apps/src/sites/code.org/pages/public/teacher-dashboard/index.js

```diff
@@ -221,6 +221,7 @@ function main() {
 
```

Comment:
earlier in this file, are we convinced that the relative path will always be relative to the right thing?
```
          allStudents: {method:'GET', url:'v2/sections/:id/students', isArray: true},
```